squid-mixin/config.libsonnet

```diff
@@ -0,0 +1,13 @@
+{
+  _config+:: {
+    dashboardTags: ['squid'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+
+    // alerts thresholds
+    alertsCriticalHighPercentageRequestErrors: 5,
```

Comment:
Are you sure you don't want a unique threshold for each alert?

Reply:
I mean I can, I thought that since alot of these alert values were the same in the spec doc, it would be more efficient to have them under one generic title. That way even in the event that one of the alert values changes, it would still be a pretty quick fix. However, I'm not really tied down to this idea. If it makes more sense to add in the others I can surely do that